### DISCUSSION TITLE
New User - search to create a playlists is confusing to me

### DISCUSSION THREAD

**Discussion by hawkiboy** [quodlibet/quodlibet#3719] [Q&A] [answered]
My understanding is that in order to create a 'smart playlist' you just do a sophisticated search with all kinds of amazing things possible, but....are there some additional examples of these? The documentation is pretty limited and it is amazing how little exists elsewhere. Maybe I am not searching on the right keywords. 

In the documentation it states: To search a specific tag, use a search like:
artist = delerium
album = bargainville
But how do you put them on separate lines? 

Being a newbie novice I don't understand when it states "The search terms can’t use quotes", then it goes on to suggest : "If you want an exact match, use quotes"

Anyway, if I could just see some actual use cases I think I could figure it out. 

For example, how do you create a playlist that will find 
-all songs in a genre
-not played in the last 5 days
-with a particular value in the comment field (for example '****')

My weak attempt:
genre="latin jazz" 
~#lastplayed>96:59
comment includes "****"

And can you create the search terms elsewhere and import them?

**Comment by declension** [comment, marked as answer]
Hi

I assume you mean the [searching section of the user guide](https://quodlibet.readthedocs.io/en/latest/guide/searching.html).

> In the documentation it states: To search a specific tag, use a search like:
artist = delerium
album = bargainville
But how do you put them on separate lines?

Ah, I see now that could be confusing. Those are two _separate_ examples of queries you could use. There is no multiline query ability.

>  "The search terms can’t use quotes", 

Yes this means that you can't _search_ for something with quotes in it like that e.g. `Nat "King" Cole`

> then it goes on to suggest : "If you want an exact match, use quotes" 

For example `genre=Metal` will match `Metal` and `Thrash Metal` and `Nu-Metal` but `genre='Metal'` will only match `Metal` (well, or `metal` etc)

> My weak attempt:
> genre="latin jazz"

Yes 👍

> ~#lastplayed>96:59

Try `#(lastplayed < 5 days)`

> comment includes "****"

Try `comment=****`

> And can you create the search terms elsewhere and import them?

Not sure what you mean here. But you can save searches, yes. Creating a playlist from a search is easiest by running a search and then
* Drag-and-dropping it onto another window with playlists browser, or
* select all, right click, _playlists_, _new playlist_

**Reply by hawkiboy** [reply]
Thank you for your kind reply. I am wondering how you combine the 3 search terms in our example? I am entering them in directly in the search bar.

**Reply by declension** [reply]
See [combining searches and negation](https://quodlibet.readthedocs.io/en/latest/guide/searching.html#combining-searches-and-negation) and [combining tag searches](https://quodlibet.readthedocs.io/en/latest/guide/searching.html#combining-tag-searches)

In your case assuming you mean combining by conjunction (this `AND` that)

e.g.
```
&(genre="latin jazz", comment=***, #(lastplayed < 5 days))
```
